### PR TITLE
Update UML diagrams for DG

### PR DESCRIPTION
**Pull request by zenatrick**

### CODE REVIEW

File: src/main/java/team/easytravel/logic/parser/fixedexpense/EditFixedExpenseCommandParser.java [hunk 1]
```diff
@@ -43,30 +44,30 @@ public EditFixedExpenseCommand parse(String args) throws ParseException {
...
                 new EditFixedExpenseCommand.EditFixedExpenseDescriptor();
 
-        if (argMultimap.getValue(PREFIX_AMOUNT).isPresent()) {
+        if (argMultimap.getValue(PREFIX_EXPENSE_AMOUNT).isPresent()) {
 
-            if (argMultimap.getValue(PREFIX_CURRENCY).isEmpty()) {
+            if (argMultimap.getValue(PREFIX_EXPENSE_CURRENCY).isEmpty()) {
                 throw new ParseException(EditFixedExpenseCommand.MESSAGE_CURRENCY_NOT_PRESENT);
-            } else if (argMultimap.getValue(PREFIX_CURRENCY).get().toLowerCase().equals("sgd")) {
+            } else if (argMultimap.getValue(PREFIX_EXPENSE_CURRENCY).get().toLowerCase().equals("sgd")) {
```
Comment (loycatherine) [inline src/main/java/team/easytravel/logic/parser/fixedexpense/EditFixedExpenseCommandParser.java:51]: Can just write .ignoreCase() instead. Can follow the code quality

File: src/main/java/team/easytravel/logic/parser/fixedexpense/EditFixedExpenseCommandParser.java [hunk 2]
```diff
@@ -43,30 +44,30 @@ public EditFixedExpenseCommand parse(String args) throws ParseException {
...
                 editFixedExpenseDescriptor.setAmount(ParserUtil
-                        .parseAmount(argMultimap.getValue(PREFIX_AMOUNT).get()));
-            } else if (argMultimap.getValue(PREFIX_CURRENCY).get().toLowerCase().equals("other")) {
+                        .parseAmount(argMultimap.getValue(PREFIX_EXPENSE_AMOUNT).get()));
+            } else if (argMultimap.getValue(PREFIX_EXPENSE_CURRENCY).get().toLowerCase().equals("other")) {
```
Comment (loycatherine) [inline src/main/java/team/easytravel/logic/parser/fixedexpense/EditFixedExpenseCommandParser.java:54]: Same as the above

File: src/main/java/team/easytravel/logic/parser/fixedexpense/EditFixedExpenseCommandParser.java (hunk 1 above)
Comment (joshlay96) [reply src/main/java/team/easytravel/logic/parser/fixedexpense/EditFixedExpenseCommandParser.java:51]: Yeah I've changed it alr in my PR. Thanks for pointing it out tho

File: src/main/java/team/easytravel/logic/parser/fixedexpense/EditFixedExpenseCommandParser.java (hunk 2 above)
Comment (joshlay96) [reply src/main/java/team/easytravel/logic/parser/fixedexpense/EditFixedExpenseCommandParser.java:54]: Same as above

**Review by loycatherine — APPROVED** [review]
Looks alright

**Pull request merged**